index.html

```diff
@@ -0,0 +1,189 @@
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <h2 class="center">About me</h2>
+      <p class="center">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="cont">
+        <button class="see center-button">Get My Resume</button>
+      </div>
+    </section>
+    <section>
+      <div class="lang img-holder">
+        <div class="circle"></div>
+        <div class="topic">Languages</div>
+        <div class="row white">
+          <div class="but2"><span class="tab">JavaScript</span></div>
+          <div class="but2"><span class="tab">Ruby</span></div>
+          <div class="but2"><span class="tab">HTML</span></div>
+          <div class="but2"><span class="tab">CSS</span></div>
+        </div>
+      </div>
+
+      <div class="lang img-holder">
+        <div class="circle"></div>
+        <div class="topic">Frameworks</div>
+        <div class="row white">
+          <div class="but2"><span class="tab">Bootstrap</span></div>
+          <div class="but2"><span class="tab">Ruby on rails</span></div>
+          <div class="but2"><span class="tab">RSpec</span></div>
+          <div class="but2"><span class="tab">Capybara</span></div>
+          <div class="but2"><span class="tab">Selenium</span></div>
+        </div>
+      </div>
+
+      <div class="lang img-holder">
+        <div class="circle"></div>
+        <div class="topic">Skills</div>
+        <div class="row white">
+          <div class="but2"><span class="tab">Codekit</span></div>
+          <div class="but2"><span class="tab">GitHub</span></div>
+          <div class="but2"><span class="tab">Codepen</span></div>
+          <div class="but2"><span class="tab">Gitlab</span></div>
+          <div class="but2"><span class="tab">Terminal</span></div>
+        </div>
+      </div>  
+    </section>
+    <section>
+      <h2 class="contact center">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      <form action="">
+        <input id="name" type="text" required placeholder="Full name" />
+        <input id="email" type="email" required placeholder="Email address" />
+        <textarea name="suggestions" placeholder="Enter text here" rows="4"></textarea></label>
+        <input class= "see" id="submit" type="Submit" value="Get in touch">
+      </form>
+      <div class="socials">
+        <i class="fab fa-brands fa-twitter"></i>
+        <i class="fab fab-brands fa-linkedin-in"></i>
+        <i class="fab fa-brands fa-github"></i>
+        <i class="fa fa-regular fa-hand-peace"></i>
+      </div>
+      <p class="center">harriet.oteng@gmail.com</p>
+      <hr class="black">
+    </section>
+    
```

Comment:
- [ ] [Optional] If possible you can remove this because you will do this next week `contact-form`.

Reply:
completed